gamemaker.py

```diff
@@ -30,4 +32,52 @@ def logout(self):
         return return_value
 
     def display_menu(self):
-        return "Options\n\nlog out\ndisplay status\n"
+        return "Options\n\nmake team\n\nlog out\ndisplay status\n"
+
+    def make_team(self, input):
+        input = input.split(" ")
+        if len(input) == 2:
+            name = input[0]
+            password = input[1]
+            self.database.add_team(team.Team(name, password, self.database))
+            ret_string = "Added " + name
+        else:
+            ret_string = "Invalid input!"
+        return ret_string
+
+    def edit_team(self, input):
+        found = False
+        input = input.split(" ")
+        if len(input) == 3:
+            list = self.database.get_teams()
+            for teams in list:
```

Comment:
Use "team" instead of "teams" since each object you're working with is a single team